apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js

```diff
@@ -3,25 +3,96 @@
 import * as React from 'react';
 import { MapView } from '@kiwicom/margarita-map';
 import { StyleSheet } from '@kiwicom/universal-components';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { getCenter } from 'geolib';
+import memoize from 'memoize-one';
+import stringify from 'json-stable-stringify';
```

Comment:
I think I read about this somewhere on Slack, right? This is to make sure the stringified version is always ordered in the same way, right?

Reply:
yes [docs here](https://github.com/substack/json-stable-stringify#json-stable-stringify)